lib/JsModule.js

```diff
@@ -201,7 +201,10 @@ export default class JsModule {
       defaultImport,
       importFunction: config.get('importFunction', { pathToImportedModule }),
       namedImports,
-      path: this.importPath,
+      path: config.get('moduleSpecifier', {
+        pathToImportedModule,
+        moduleSpecifier: this.importPath,
+      }),
```

Comment:
That turned out to be non-trivial. We use `importPath` in the public API that the editor plugins use, and I'd rather push the rename to a different time as it requires updates to all the plugins as well. I could of course rename it locally only, but I'd rather keep a funky but consistent name than a more correct but inconsistent one.